internal/healthchecks/error.go

```diff
@@ -164,6 +165,22 @@ var (
 		ResourceLink: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/authorization",
 		IsFatal:      true,
 	}
+	LogPipelineErr = HealthCheckError{
+		Code:         "LogPipelineErr",
+		Class:        Runtime,
+		Message:      "Ops Agent logging pipeline failed",
+		Action:       "Refer to provided documentation link.",
+		ResourceLink: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/troubleshoot-find-info",
+		IsFatal:      true,
+	}
+	LogParseErr = HealthCheckError{
+		Code:         "LogParseErr",
+		Class:        Runtime,
+		Message:      "Ops Agent failed to parse logs",
+		Action:       "Refer to provided documentation link.",
+		ResourceLink: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/troubleshoot-find-info",
+		IsFatal:      true,
```

Comment:
Is this actually a fatal error? Won't fluent-bit continue to run if it fails to parse a log?

Reply:
It is not a fatal error. Thanks for mentioning. I changed to `IsFatal: false`.